src/ReinforcementLearningBase/src/interface.jl

```diff
@@ -404,6 +404,15 @@ Make an independent copy of `env`,
 @api copy(env::AbstractEnv) = deepcopy(env)
 @api copyto!(dest::AbstractEnv, src::AbstractEnv)
 
+# checking the state of all players in env is enough?
+"""
+    Base.:(==)(env1::T, env2::T) where T<:AbstractEnv
+!!! warning: Only check the state of all players in the env.
+"""
+Base.:(==)(env1::T, env2::T) where T<:AbstractEnv = 
+    sum([state(env1, player) == state(env2, player) for player in players(env1)]) == length(players(env1))
```

Comment:
Use `all` instead of `sum` here. And the list comprehension is not necessary, an iterator will be enough.

Reply:
Thank you for your responding and I have pushed changes to the branch.